### PR TITLE
Change to more tick efficient solution

### PR DESCRIPTION
**Pull request by PowContent**
The provided solution is optimised for tick efficiency given a start tile by the tablet and a end tile at the door. Sanity checked, values add up when tested against a lookup array:
```[9, 3, 7, 2, 9, 7, 6, 4, 1, 1, 8, 6, 3, 8, 5, 6, 2, 7, 2, 4, 1, 9, 5, 4, 3]```

### CODE REVIEW

**Comment by PowContent** [comment]
[kephri solutions.zip](https://github.com/LlemonDuck/tombs-of-amascut/files/10378121/kephri.solutions.zip)
All Optimal Solutions by number

File: src/main/java/com/duckblade/osrs/toa/features/scabaras/overlay/AdditionPuzzleSolver.java [hunk 1]
```diff
@@ -75,31 +75,31 @@ public class AdditionPuzzleSolver implements PluginLifecycleComponent
...
 		.put(37, ImmutableSet.of(10, 11, 12, 13, 14, 19, 24))
-		.put(38, ImmutableSet.of(2, 8, 10, 11, 12, 13, 14))
-		.put(39, ImmutableSet.of(0, 1, 2, 3, 4, 9, 13))
-		.put(40, ImmutableSet.of(0, 5, 10, 15, 20, 21))
-		.put(41, ImmutableSet.of(0, 1, 2, 5, 10, 15, 20))
-		.put(42, ImmutableSet.of(0, 6, 12, 18, 20, 21, 22, 23, 24))
-		.put(43, ImmutableSet.of(0, 1, 2, 3, 5, 10, 15, 20))
-		.put(44, ImmutableSet.of(1, 2, 3, 4, 6, 11, 16, 21))
-		.put(45, ImmutableSet.of(0, 5, 10, 15, 16, 17, 18, 19))
+		.put(38, ImmutableSet.of(0, 5, 6, 12, 18, 24))
```
Comment (LlemonDuck) [inline src/main/java/com/duckblade/osrs/toa/features/scabaras/overlay/AdditionPuzzleSolver.java:95]: I believe this one is missing 10. Can you confirm?

image

**Review by LlemonDuck — CHANGES_REQUESTED** [review]
Just the one solution needs a second look.

Since (I expect) you're making a change for that, can you also make sure the sets use increasing order for their values?

**Comment by LlemonDuck** [comment]
added in daadc1d1b1c2d7f7a0ac932c89e183b4af2ef681

**Pull request closed without merging**